product_contract/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Product Contract',
+    'version': '10.0.1.0.0',
+    'category': 'Contract Management',
+    'license': 'AGPL-3',
+    'author': "LasLabs, "
+              "Odoo Community Association (OCA)",
+    'website': 'https://laslabs.com',
+    'depends': [
+        'base',
```

Comment:
Only time you need to depend on base is when you don't have any other depends. One can assume that all modules follow this principle, so at some point you will hit a base depends once you hit the bottom of the depends stack.